PivotController/PivotController/Controllers/PivotController.cs

```diff
@@ -36,6 +37,34 @@ public async Task<object> Post([FromBody]object args)
             {
                 return await GetRawData(param);
             }
+            else if (param.Action == "onExcelExport" || param.Action == "onCsvExport")
+            {
+                if (param.InternalProperties.EnableVirtualization && param.InternalProperties.ExportAllPages)
+                {
+                    EngineProperties engine = await GetEngine(param);
+                    engine = await PivotEngine.PerformAction(engine, param);

```

Comment:
provide if condition for this line alone.

Reply:
Resolved